osler/inventory/tests.py

```diff
@@ -0,0 +1,110 @@
+from django.test import TestCase
+from django.urls import reverse, resolve
+from osler.inventory.models import DrugCategory, MeasuringUnit, Manufacturer, Drug
+from osler.inventory import views, forms
+# Create your tests here.
+
+class TestUrls(TestCase):
+
+    def test_drug_list_url(self):
+        path = reverse('inventory:drug-list')
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-list')
+
+    def test_drug_add_new_url(self):
+        path = reverse('inventory:drug-add-new')
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-add-new')
+
+    def test_pre_add_new_drug_url(self):
+        path = reverse('inventory:pre-drug-add-new')
+        self.assertEqual(resolve(path).view_name, 'inventory:pre-drug-add-new')
+
+    def test_predrug_select_url(self):
+        path = reverse('inventory:predrug-select')
+        self.assertEqual(resolve(path).view_name, 'inventory:predrug-select')
+
+    def test_drug_dispense_url(self):
+        path = reverse('inventory:drug-dispense')
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-dispense')
+
+    def test_drugupdate_url(self):
+        path = reverse('inventory:drug-update', kwargs={'pk':1})
+        self.assertEqual(resolve(path).view_name, 'inventory:drug-update')
+
+def drug_dict():
+
+    drug_dict = {
+        'name': 'Somedrug',
+        'unit': MeasuringUnit.objects.create(name='someunit'),
+        'dose': 1000.0,
+        'stock': 10,
+        'expiration_date': '2040-01-01',
+        'lot_number': 'ABCDEFGH',
+        'category': DrugCategory.objects.create(name='somecategory'),
+        'manufacturer': Manufacturer.objects.create(name='somemanufacturer')
+    }
+
+    return drug_dict
+
+class TestModels(TestCase):
+
```

Comment:
I agree that when I create a drug, I have to create all these other fields as well. But, these "unit tests" are helpful in that they catch which model failed to create an object, whereas getting an error from creating a drug leaves us clueless whether the manufacturer, measuring unit, drug category, or the drug itself went wrong. I agree with you that they may be redundant, but I would like to keep it since it will help me pin point specifically which of the four models went wrong when creating an object.